linux/check_quiesce_snapshot_custom_script/check_quiesce_snapshot_custom_script.yml

```diff
@@ -50,7 +50,7 @@
             - pre-freeze-script
             - post-thaw-script
           delegate_to: "{{ vm_guest_ip }}"
-          when: "'Flatcar' not in guest_os_ansible_distribution"
+          when: "'Flatcar' != guest_os_ansible_distribution"
```

Comment:
Please update to `guest_os_ansible_distribution != 'Flatcar'`, and so as the following changes.